ipn/ipnlocal/local.go

```diff
@@ -1188,6 +1192,37 @@ func (b *LocalBackend) SetControlClientStatus(c controlclient.Client, st control
 	b.authReconfig()
 }
 
+func applySysPolicy(prefs *ipn.Prefs) (anyChange bool) {
```

Comment:
This should also include `ExitNodeAllowLANAccess`.

Reply:
This PR is strictly for moving the existing items out of `defaultPrefs`. I'll add the missing ones in a subsequent PR.

Reply:
`ExitNodeAllowLANAccess` is queued up to go in immediatley after this PR https://github.com/tailscale/tailscale/compare/adrian/enforce-syspolicy...adrian/additional-syspolicy